.github/policies/BreakingChangeManagement.yml

```diff
@@ -0,0 +1,43 @@
+id: 
```

Comment:
This file isn't in testfx. Is it a copy from somewhere?

Reply:
Yes, this is dotnet/sdk integrated repos policy, you can find it in the resource management yml file that was in this repo. I removed the rules from there because they are largely duplicated with what this PR is bringing in, except for this rule.